pcdsdevices/mv_interface.py

```diff
@@ -4,6 +4,9 @@
 import time
 import fcntl
 import logging
+import threading
+from threading import Thread
```

Comment:
This should be rewritten to either be:
`import threading`
or
`from threading import Thread, Event`

Instead of doing it both ways, for consistency's sake. You may have to change how this is used elsewhere in the file.